contrib/email-notifications/report-configsnap.post-apply

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [[ -f /root/auter-configsnap-$(date +%Y-%m-%d)/configsnap/post-update.compare ]]; then
```

Comment:
This entirely depends on configsnap so that should be included in the dependency list. You should also expand your report-configsnap.post-apply section in README.md to include a note that this depends on the specific naming convention for the configsnap `-d` option. You have used the default config from the current configsnap contrib script but this is a customizable option so should be stated in the README.